docs/modules/ref-tssc-workflow-components.adoc

```diff
@@ -255,52 +255,52 @@ Detect new, changed, or merged,  Merge Request to Release Branch
 * Source Control Tool
 
 | Deploy or Update TEST Environment
-| #TODO - purpose#
+| Deploy code to shared test environment with latest feature available for shared testing
```

Comment:
Suggestion:  
Deploy image built from the latest release branch to the test environment

Reply:
Updated as suggested